finished-proposals.md

```diff
@@ -59,7 +59,7 @@ Finished proposals are proposals that have reached stage 4, and are included in
 | [RegExp v flag with set notation + properties of strings][regexp-v-flag] | Markus Scherer<br />Mathias Bynens                     | Mathias Bynens                                          | [May 2023][regexp-v-flag-notes]             | 2024                      |
 | [Resizable and growable ArrayBuffers][resizable]                         | Shu-yu Guo                                             | Shu-yu Guo                                              | [September 2023][resizable-notes]           | 2024                      |
 | [Array Grouping][array-grouping]                                         | Justin Ridgewell                                       | Justin Ridgewell<br />Jordan Harband                    | [November 2023][array-grouping-notes]       | 2024                      |
-| [`Promise.withResolvers`][promise-defer]                                 | Peter Klecha                                           | Peter Klecha                                            | [November 2023][promise-defer-notes]        | 2024                      |
+| [`Promise.withResolvers`][promise-defer]                                 | Peter Klecha                                           | Peter Klecha                                            | [November 2023][promise-defer-notes-2023-11]<br>[July 2023][promise-defer-notes-2023-07]<br>[May 2023][promise-defer-notes-2023-05]<br>[March 2023][promise-defer-notes-2023-03]       | 2024                      |
```

Comment:
```suggestion
| [`Promise.withResolvers`][promise-defer]                                 | Peter Klecha                                           | Peter Klecha                                            | <sub><ul><li>[November 2023][promise-defer-notes-2023-11]</li><li>[July 2023][promise-defer-notes-2023-07]</li><li>[May 2023][promise-defer-notes-2023-05]</li><li>[March 2023][promise-defer-notes-2023-03]</li></ul></sub>       | 2024                      |
```
maybe?

Reply:
overlooking the fact that that is invalid HTML, the rendering leaves something to be desired...

![Screenshot 2024-02-14 at 2 26 44 PM](https://github.com/tc39/proposals/assets/15326526/94e8cf3e-b36a-4250-8486-cdc7f7513033)

Reply:
oof, yeah. maybe putting it inside a `<details>`?

Reply:
not possible